safe/gui/tools/osm_downloader_dialog.py

```diff
@@ -401,6 +401,11 @@ def download(self, feature_type):
         if output_prefix is not None:
             url += '&output_prefix=%s' % output_prefix
 
+        if 'LANG' in os.environ:
```

Comment:
See also https://github.com/AIFDR/inasafe/blob/develop/__init__.py - LANG may have already be split using '_'